pkg/pdfcpu/attach.go

```diff
@@ -197,57 +197,55 @@ func removeAttachedFiles(xRefTable *XRefTable, files StringSet) (ok bool, err er
 
 	log.Debug.Println("removeAttachedFiles begin")
 
-	if len(files) > 0 {
-
-		var removed bool
+	// If no files specified, remove all embedded files.
+	if len(files) == 0 {
+		err = xRefTable.RemoveEmbeddedFilesNameTree()
+		if err != nil {
+			return false, err
+		}
+		return true, nil
+	}
 
-		for fileName := range files {
+	removed := false
```

Comment:
var removed bool .. initializes to false
I would like to keep this here.

Reply:
Ok, as you wish!